.gitignore

```diff
@@ -23,3 +23,6 @@ tsconfig.tsbuildinfo
 
 # compiled
 **/tailwind-output.css
+
+# module-manager archives
+packages/module-manager/archive/**.*
```

Comment:
It need to be on repo, because when somebody have removed 1 module, somebody else will need to be able to recover it.